tasks/messages/message_test.go

```diff
@@ -106,13 +106,15 @@ func TestParseMessageParams(t *testing.T) {
 		{
 			// Derived from message bafy2bzacebpiuu7tgya6yz56sfllpqc3rqbo5s5xl7353xeuavc53qlpb4sqw
 			// Account actor is supported for parameter parsing it the constructor method.
+			// expect error since parameters we not passed but expected:
+			// https://filfox.info/en/message/bafy2bzacebpiuu7tgya6yz56sfllpqc3rqbo5s5xl7353xeuavc53qlpb4sqw
 			name:        "issue-709",
 			method:      1,
 			params:      nil,
 			actorCode:   builtin3.AccountActorCodeID,
 			wantMethod:  "Constructor",
-			wantEncoded: `""`,
-			wantErr:     false,
+			wantEncoded: ``,
+			wantErr:     true,
```

Comment:
I introduced this mistake here: https://github.com/filecoin-project/lily/pull/714/files#diff-cbf6199b767dd5286dea0dea57200b50ead0e919d0079734bc3e0cc2bb006fc1R114.
An error is expected.